pkg/registry/channelupdateoptions.go

```diff
@@ -0,0 +1,26 @@
+package registry
+
+import (
+	"strings"
+)
+
+type Mode int
+
+const (
+	ReplacesMode = iota
+	SemVerMode
+	SkipPatchMode
+)
+
+func GetModeFromString(mode string) Mode {
+	switch strings.ToLower(mode) {
+	case "replaces":
+		return ReplacesMode
+	case "semver":
+		return SemVerMode
+	case "semver-skippatch":
+		return SkipPatchMode
+	default:
```

Comment:
I sort of disagree, because just not setting it gives you default behavior you describe. If you type `opm index add --mode=seemver` and it just attempts to insert in replaces mode I would be very confused.